app/script/view_model/content/InputBarViewModel.js

```diff
@@ -296,6 +297,9 @@ z.viewModel.content.InputBarViewModel = class InputBarViewModel {
       this.editMessageEntity(messageEntity);
       this.editMessageEntity().isEditing(true);
       this.input(this.editMessageEntity().get_first_asset().text);
+      this.currentMentions = this.editMessageEntity()
+        .get_first_asset()
+        .mentions();
       if (inputElement) {
```

Comment:
I suppose we can simplify things a bit if we apply the state to the plain `messageEntity` - instead of getting it from the observable every time.

````
messageEntity.isEditing(true);
this.input(messageEntity.get_first_asset().text);
this.currentMentions = messageEntity.get_first_asset().mentions();
this.editMessageEntity(messageEntity);
````